src/xdocs/config_javadoc.xml

```diff
@@ -154,6 +154,58 @@
       </subsection>
     </section>
 
+    <section name="InvalidJavadocPosition">
+      <p>Since Checkstyle 8.21</p>
+      <subsection name="Description" id="InvalidJavadocPosition_Description">
+        <p>
+          Checks that Javadocs are located at the correct position.
```

Comment:
Please add the link to the specification:
https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html

Reply:
Done.